builder/dispatchers.go

```diff
@@ -53,7 +53,6 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 		// name  ==> args[j]
 		// value ==> args[j+1]
 		newVar := args[j] + "=" + args[j+1] + ""
-		commitStr += " " + newVar
```

Comment:
just checked - and there are no tests of the cache w.r.t. LABELs.  scary!  :-)   will work on that.